dev/AppLifecycle/ActivatedEventArgsBase.h

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#pragma once
+
+namespace winrt::Microsoft::ProjectReunion::implementation
+{
+    using namespace winrt::Windows::ApplicationModel::Activation;
+
+    class ActivatedEventArgsBase : public winrt::implements<ActivatedEventArgsBase,
+        IActivatedEventArgs>
+    {
+    public:
+        // IActivatedEventArgs
+        ActivationKind Kind()
+        {
+            return m_kind;
+        }
+
+        ApplicationExecutionState PreviousExecutionState()
+        {
+            return m_previousState;
+        }
+
+        SplashScreen SplashScreen()
+        {
+            return m_splashScreen;
+        }
+
+    protected:
+        ActivatedEventArgsBase()
+        {
+        }
+
+        ActivationKind m_kind;
+        ApplicationExecutionState m_previousState;
```

Comment:
does this have a reasonable initial value?

Reply:
Updated